dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/model/FromInstruction.java

```diff
@@ -190,6 +191,16 @@ public boolean hasADifferentTag(String expectedTag) {
         return !tag.trim().equals(expectedTag.trim());
     }
 
+    /**
+     * Determines whether the comment has mentioned "no-dfiu" to ignore creating dfiu PR
+     */
+    public boolean hasCommentNoDfiu() {
```

Comment:
Perhaps reverse the boolean logic and call this `shouldNotIgnore()`?

Thoughts @afalko ?

Reply:
Thanks @justinharringa for reviewing. Here is my take on this:
According to me current name `hasCommentNoDfiu()` and purpose is more readable.
Still i am open for change to as below:
```
public boolean shouldNotIgnorePR() {
        if (hasComments()) {
            return !comments.contains(NO_DFIU);
        }
        return true;
    }
```
Kindly confirm if you feel this is more appropriate.